Analysis/Tasks/correlations.cxx

```diff
@@ -53,6 +47,15 @@ struct CorrelationTask {
   O2_DEFINE_CONFIGURABLE(cfgPairCutPhi, float, -1, "Pair cut on Phi: -1 = off; >0 otherwise distance value")
   O2_DEFINE_CONFIGURABLE(cfgPairCutRho, float, -1, "Pair cut on Rho: -1 = off; >0 otherwise distance value")
 
+  // Filters and input definitions
+  Filter trackFilter = (aod::track::eta > -cfgCutEta) && (aod::track::eta < cfgCutEta) && (aod::track::pt > cfgCutPt) && (aod::track::isGlobalTrack == (uint8_t)1);
```

Comment:
Does `-cfgCutEta` work? I'm not sure how it will be parsed, it may be implicitly converted to a float and become a literal instead of placeholder. Have you checked if it is properly changed when you change it in json?

Reply:
Thanks - indeed it didn't work. I have changed it.